maven-surefire-plugin/src/site/apt/history.apt.vm

```diff
@@ -0,0 +1,90 @@
+*---------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                             | Test Report tasks (prerequisite: SUREFIRE-1222 in 3.0.0-M4)                                                                                                                       \ |
+|                                             | New interprocess communication with TCP/IP which fixed current blocker and critical bugs.                                                                                         \ |
+| 3.0.0-M5                                    | Provided extensions which can be used to customize this interprocess communication in plugin configuration. It is internally used to switch over the pipes and TCP connector.     \ |
+|                                             |                                                                                                                                                                                   \ |
+|                                             | {{{https://issues.apache.org/jira/browse/SUREFIRE-1658}TCP/IP Channel for forked Surefire JVM. Extensions API and SPI. Polymorphism for remote and local process communication.}} \ |
+*---------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 3.0.0-M6                                    | ReportEntry contains new fields testRunId:long and RunMode which help identifying the tests and logs. The plugin supports Java 1.8 and Maven Plugin API 3.2.5. Many bug fixes.    \ |
+|                                             | {{{https://issues.apache.org/jira/issues/?jql=project%20%3D%20SUREFIRE%20AND%20fixVersion%20%3D%203.0.0-M6%20ORDER%20BY%20priority%20DESC%2C%20key}ASF Release versions}}         \ |
+*---------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 3.0.0-M7                                    | SurefireProvider interface, TestSetRunListener, StatelessXmlReporter, and related code will be reworked. Currently, Java classes are executed as tests and we will support more.  \ |
+|                                             | {{{https://issues.apache.org/jira/issues/?jql=project%20%3D%20SUREFIRE%20AND%20fixVersion%20%3D%203.0.0-M7%20ORDER%20BY%20priority%20DESC%2C%20key}ASF Release versions}}         \ |
```

Comment:
Does this description is correct according to planning release?

Reply:
I copy/paste content from other page so preserving content